src/renate/memory/storage.py

```diff
@@ -50,9 +61,16 @@ class MemoryMappedTensorStorage(Storage):
         length: Number of items to be stored.
     """
 
-    def __init__(self, directory: str, data_point: NestedTensors, length: int) -> None:
-        super().__init__(directory, data_point, length)
-        self._storage = self._create_mmap_tensors(directory, data_point, length)
+    def __init__(self, directory: str) -> None:
```

Comment:
why do we keep this class?

Reply:
Purely for reference. I suggest we keep this class even though its not used anywhere.